CoreVisual/src/au/gov/asd/tac/constellation/visual/icons/FlagIconProvider.java

```diff
@@ -28,883 +28,883 @@
 @ServiceProvider(service = ConstellationIconProvider.class)
 public class FlagIconProvider implements ConstellationIconProvider {
 
-    public static final ConstellationIcon AFGHANISTAN = new ConstellationIcon.Builder("Afghanistan", new FileIconData("modules/ext/icons/afghanistan.png", "au.gov.asd.tac.constellation.visual"))
+    public static final ConstellationIcon AFGHANISTAN = new ConstellationIcon.Builder("Afghanistan", new FileIconData("modules/ext/icons/flags/afghanistan.png", "au.gov.asd.tac.constellation.visual"))
```

Comment:
Thanks @sol695510, creating the flags folder was something I wanted to see done too. Nice 👍